python/ray/data/read_api.py

```diff
@@ -698,6 +698,124 @@ def read_parquet(
     )
 
 
+@PublicAPI
```

Comment:
`@PublicAPI(stability="alpha")`

Reply:
Thanks for the review @c21 ! We're under a discussion about the correct way to support this, as it seems delta-rs is not perfectly ready for this integration. So please feel free to ignore this PR for now :) Appreciate your time!

Reply:
cool, SG. Take your time.